utest/parsing/test_model.py

```diff
@@ -648,10 +648,17 @@ def test_invalid(self):
         assert_model(node, expected)
 
 
+class RemoveNonDataTokensVisitor(ModelVisitor):
+    def visit_Statement(self, node):
+        node.tokens = node.data_tokens
```

Comment:
Clever way to be able to use `data_only=True` without needing to specify all separators and EOLs in the expected model.